### PR TITLE
chore: improve clickhouse schema

### PR DESCRIPTION
**Pull request by maxdeichmann**
> [!IMPORTANT]
> Enhance ClickHouse schema with new columns, tables, views, and a seeding script, and improve code style in `IngestionService`.
> 
>   - **Schema Changes**:
>     - Add `event_ts` column to `traces`, `observations`, and `scores` tables.
>     - Modify `ORDER BY` clause in `traces`, `observations`, and `scores` to use `toDate()`.
>     - Add `time_to_first_token` column to `observations` table.
>     - Introduce `observations_wide` table with materialized views `mv_traces_to_observations_wide` and `mv_observations_to_observations_wide`.
>   - **Scripts**:
>     - Add `seed.ts` script to populate ClickHouse tables with sample data.
>     - Update `package.json` to include `ch:seed` script.
>   - **Code Style**:
>     - Remove trailing commas in `IngestionService` class methods in `index.ts`.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=langfuse%2Flangfuse&utm_source=github&utm_medium=referral)<sup> for 3d369f63f91c25f6ed057584db90c609fd0cd68e. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

**Review by Steffen911 — APPROVED** [review]

File: packages/shared/clickhouse/migrations/0001_traces.up.sql [hunk 1]
```diff
@@ -15,12 +15,13 @@ CREATE TABLE traces (
     `session_id` Nullable(String),
     `created_at` DateTime64(3) DEFAULT now(),
     `updated_at` DateTime64(3) DEFAULT now(),
+    event_ts DateTime64(3),
```
[the remark this answers is not in the document]
Comment (maxdeichmann) [reply packages/shared/clickhouse/migrations/0001_traces.up.sql:18]: Yes, we always have this value.

File: packages/shared/clickhouse/migrations/0002_observations.up.sql [hunk 2]
```diff
@@ -30,11 +30,13 @@ CREATE TABLE observations (
     `output_cost` Nullable(Decimal64(12)),
     `total_cost` Nullable(Decimal64(12)),
     `completion_start_time` Nullable(DateTime64(3)),
+    `time_to_first_token` Nullable(DateTime64(3)),
```
[the remark this answers is not in the document]
Comment (maxdeichmann) [reply packages/shared/clickhouse/migrations/0002_observations.up.sql:33]: Good catch!

File: worker/src/services/IngestionService/index.ts [hunk 3]
```diff
@@ -68,7 +68,7 @@ export class IngestionService {
     private redis: Redis,
     prisma: PrismaClient,
     private clickHouseWriter: ClickhouseWriter,
-    private clickhouseClient: ClickhouseClientType,
+    private clickhouseClient: ClickhouseClientType
```
[the remark this answers is not in the document]
Comment (maxdeichmann) [reply worker/src/services/IngestionService/index.ts:71]: Yes, i think the issue is on my end.

File: packages/shared/clickhouse/migrations/0002_observations.up.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (maxdeichmann) [reply packages/shared/clickhouse/migrations/0002_observations.up.sql]: Actually, i will revert this entirely. This is calculated by: start_time - completion_start_time

**Pull request merged**